src/main/java/com/mcmiddleearth/guidebook/command/GuidebookCommand.java

```diff
@@ -55,6 +57,17 @@ public void handle(CommandSender cs, String... args) {
 
     protected abstract void execute(CommandSender cs, String... args);
 
+
+    /**
+     * @param cs   The sender
+     * @param args The arguments provided for this subcommand - <code>/guidebook subcommand arg0 arg1<code>
+     * @return By default, an empty list
+     */
+    protected List<String> getCompletions(CommandSender cs, String... args) {
+        // Default - no completions
+        return List.of();

```

Comment:
I was thinking about having the default be the names of the areas, but it felt a bit dangerous/confusing in the case that a new command is added